x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/mapper/SemanticTextField.java

```diff
@@ -112,10 +136,12 @@ private void validate() {
                 case TEXT_EMBEDDING:
                     validateFieldPresent(DIMENSIONS_FIELD, dimensions);
                     validateFieldPresent(SIMILARITY_FIELD, similarity);
+                    validateFieldPresent(ELEMENT_TYPE_FIELD, elementType);
```

Comment:
We need to make sure that this PR is part of the first release for semantic text, otherwise some already indexed documents might miss this new mandatory type. Is it part of the plan?

Reply:
Exactly. Without this PR, Cohere byte embeddings don't work, so I consider this change essential for the first release.